### PR TITLE
Jacquit widget

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Hello Team,

I have just finished adding the widget and activating it, so when a user clicks or mouse hovers my account link, then the widget opens up, and by clicking one of the header links unlike my account one, the widget will close and the user will be navigated to another page.
Please have a look at it here Any comment would be highly appreciated.

Thanks in advance.

### CODE REVIEW

File: src/components/MyOffers/MyOffers.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        <h5>Profile:</h5>
+        <ProfileLink>
+          <li>
+            <Link to={`${Paths.PROFILE}/:${username}`}>Preview</Link>
+          </li>
+          <li>
+            <Link to={`${Paths.PROFILE}/:${id}`}>Edit</Link>
+          </li>
+        </ProfileLink>
+      </Profile>
```
Comment (andrzej-onja) [inline src/components/MyOffers/MyOffers.tsx:47]: profile was not included in the design for this page

File: src/components/MyOffers/MyOffers.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+import addSVG from "./icons/plus-circle.svg"
+import { Offer } from '../Offer/Offer'
+import { Paths } from '../../paths'
+
+export interface Photo {
+  url: string
+  info: string
+  isPrimary: boolean
+}
+interface OfferData {
```
Comment (ganamavo) [inline src/components/MyOffers/MyOffers.tsx:15]: I'd make this a type instead

File: src/components/MyOffers/MyOffers.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import Button from '../Buttons/Buttons'
+import edit from './icons/edit.svg'
+import deleteIcon from './icons/delete.svg'
+import addSVG from "./icons/plus-circle.svg"
+import { Offer } from '../Offer/Offer'
+import { Paths } from '../../paths'
+
+export interface Photo {
```
Comment (ganamavo) [inline src/components/MyOffers/MyOffers.tsx:10]: Looks like a type

**Comment by andrzej-onja** [comment]
closing as it was a part of graded project only

**Pull request closed without merging**